deploy/helm-chart/kubernetes-replicator/templates/deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         {{- toYaml . | nindent 8 }}
     {{- end }}
       serviceAccountName: {{ include "kubernetes-replicator.serviceAccountName" . }}
+      automountServiceAccountToken: {{ include "kubernetes-replicator.automountServiceAccountToken" . }}
```

Comment:
You're calling a macro called `"kubernetes-replicator.automountServiceAccountToken"` here, which is defined nowhere. Please either define the macro or just use `.Values.automountServiceAccountToken` directly.